disnake/types/activity.py

```diff
@@ -44,8 +45,10 @@ class ActivityAssets(TypedDict, total=False):
     # https://discord.com/developers/docs/topics/gateway-events#activity-object-activity-asset-image
     large_image: str
     large_text: str
+    large_url: str
     small_image: str
     small_text: str
+    small_url: str
```

Comment:
It would be great to have these exposed as properties in `_BaseActivity`, similar to e.g. `large_image_text`. Issue is that `large_image_url` already exists for the asset url, maybe `large_image_link` would work as the name for this `large_url` field?